domain/schema/model/sql/0022-resource.sql

```diff
@@ -29,103 +29,85 @@ INSERT INTO resource_state VALUES
 
 CREATE TABLE resource (
     uuid TEXT NOT NULL PRIMARY KEY,
-    application_uuid TEXT NOT NULL,
-    name TEXT NOT NULL,
     revision INT,
     origin_type_id INT NOT NULL,
+    retrieved_by_uuid TEXT,
     state_id INT NOT NULL,
-    size INT,
-    hash TEXT,
-    hash_type_id TEXT,
     created_at TIMESTAMP NOT NULL,
-    CONSTRAINT fk_resource_application_uuid
-    FOREIGN KEY (application_uuid)
-    REFERENCES application (uuid),
-    CONSTRAINT fk_resource_name
-    FOREIGN KEY (name)
-    REFERENCES resource_meta (name),
+    -- last_polled is when the repository was last polled for new resource
+    -- revisions. Only set if resource_state is 1.
+    last_polled TIMESTAMP NOT NULL,
     CONSTRAINT fk_resource_origin_type_id
     FOREIGN KEY (origin_type_id)
     REFERENCES resource_origin_type (id),
     CONSTRAINT fk_resource_state_id
     FOREIGN KEY (state_id)
     REFERENCES resource_state (id),
-    CONSTRAINT fk_resource_hash_type_id
-    FOREIGN KEY (hash_type_id)
-    REFERENCES object_store_metadata_hash_type (id)
+    CONSTRAINT fk_resource_retrieved_by_uuid
+    FOREIGN KEY (retrieved_by_uuid)
+    REFERENCES resource_retrieved_by (uuid)
```

Comment:
**todo:* Unconsistency between resource table and resource retrieved_by (circular reference, either `resource_retrieved_by.resource_uuid => FK resource.uuid`, either the `resource.retrieved_by_uuid => FK resource_retrieved_by.uuid` not both)

However I think the former is the right one since this FK doesn't reference an actual uuid (no such field in `resource_retrieved_by`